### ISSUE TITLE
UUIDv7 logo

### ISSUE THREAD

**Issue by sergeyprokhorenko** [uuid6/uuid6-ietf-draft#130]
I encourage you to post here proposed UUIDv7 logos

**Comment by broofa** [comment]
This strikes me as a bit off-topic.

**Comment by ramsey** [comment]
What's a "UUIDv7 logo?"

**Comment by sergeyprokhorenko** [comment]
https://logos.fandom.com/wiki/Intel_Inside

**Comment by sergeyprokhorenko** [comment]
The purpose of this proposal is to enable vendors of DBMS and other platforms to advertise the presence of UUIDv7 generators in their platforms. Just like the famous Intel Inside logo: https://logos.fandom.com/wiki/Intel_Inside
![image](https://github.com/uuid6/uuid6-ietf-draft/assets/31864557/90f5b326-3de1-4dd3-839d-70c4126b4931)


As an example, I can offer this:
![image](https://github.com/uuid6/uuid6-ietf-draft/assets/31864557/89cea253-5acf-42c4-83b0-7cb3002f0f03)

**Comment by ramsey** [comment]
None of the other UUID versions have logos. Why one for v7 only?

**Comment by sergeyprokhorenko** [comment]
> None of the other UUID versions have logos. Why one for v7 only?

UUIDv7 is the most important and popular. I don't mind other versions of UUID having logos too, but those logos are unlikely to be popular.

**Comment by ben221199** [comment]
@broofa I agree it is off-topic for the spec itself, but I think it is useful for the project page.

 - CSS didn't had a logo, but it has one (official) for CSS3: https://en.wikipedia.org/wiki/CSS.
 - The WHATWG specs all have their own logo: https://resources.whatwg.org/. (See files starting with `logo-`.)
 - The Multiformats project has its own logo: https://multiformats.io/.

@sergeyprokhorenko I suggest we make a generic logo that represents UUID, the logo we can place on some dedicated UUID site. Eventually it can be designed to be extensible, so that we are also able to show the variants, versions, families and future distinction types.

**Comment by sergeyprokhorenko** [comment]
> I suggest we make a generic logo that represents UUID, the logo we can place on some dedicated UUID site. Eventually it can be designed to be extensible, so that we are also able to show the variants, versions, families and future distinction types.

![image](https://github.com/uuid6/uuid6-ietf-draft/assets/31864557/2a83bb01-46cd-4f0f-924d-ad9fb431e3b3)

**Comment by ben221199** [comment]
I got something like this, but I was unable yet to make rounded borders:
![image](https://github.com/uuid6/uuid6-ietf-draft/assets/12856904/60e42872-2b4d-4f7a-a9cb-d6ac264eabca)

**Comment by ben221199** [comment]
> > I suggest we make a generic logo that represents UUID, the logo we can place on some dedicated UUID site. Eventually it can be designed to be extensible, so that we are also able to show the variants, versions, families and future distinction types.
> 
> ![image](https://user-images.githubusercontent.com/31864557/260949855-2a83bb01-46cd-4f0f-924d-ad9fb431e3b3.png)

I like it, but I want to see one without version too, and we have to take "variants" (and "families") into account.

**Comment by sergeyprokhorenko** [comment]
> but I want to see one without version too

![image](https://github.com/uuid6/uuid6-ietf-draft/assets/31864557/3f8797d1-f21c-4c93-9802-24617a1a5df3)

**Comment by broofa** [comment]
<img width="496" alt="CleanShot 2023-08-16 at 07 41 37@2x" src="https://github.com/uuid6/uuid6-ietf-draft/assets/164050/a0f9b909-6a5f-4924-a261-32129c1f2af5">

**Comment by ramsey** [comment]
@broofa I like the two in the top row.

**Comment by sergeyprokhorenko** [comment]
It seems to me that @broofa has set the right direction: the logo should not look like a boring UML diagram. It should be more pleasant and fun. But we need to go further in this direction.

**Comment by broofa** [comment]
<img width="595" alt="CleanShot 2023-08-17 at 12 56 16@2x" src="https://github.com/uuid6/uuid6-ietf-draft/assets/164050/85314fee-d665-424d-9f3f-be86d172a1da">

**Comment by sergeyprokhorenko** [comment]
> <img alt="CleanShot 2023-08-17 at 12 56 16@2x" width="595" src="https://user-images.githubusercontent.com/164050/261422869-85314fee-d665-424d-9f3f-be86d172a1da.png">

I like it very much. And I had the same idea, so UUIDs are like a black hole around which the stars of a spiral galaxy revolve.

I would add color #AD1022 and 3D:

<img width="160" alt="UUIDv7 logo_new_2" src="https://github.com/uuid6/uuid6-ietf-draft/assets/31864557/785efc9b-97b0-482a-81eb-f9059399d8d6">

The font is bold Aptos.

**Comment by broofa** [comment]
> UUIDs are like a black hole around which the stars of a spiral galaxy revolve

I was going more for "a universe of IDs" concept but "black hole" works too, I suppose. :-)

> I would add color 

I'm just throwing roughed-out ideas around here.  Color is complex issue in logo design for a whole bunch of reasons (personal opinion, color blindness, cultural significance, print vs. screen considerations, etc.)  It's a whole other level of discussion that I've found tends to be a distraction in the initial stages of design.  And a good logo should be able to stand on it's own without color.  At the end of the day, black-and-white printers are still a thing.

Anyhow, if anyone wants to mess around with any of this, here's my current source document (SVG, open in [Inkscape](https://inkscape.org/)):

<img src="https://github.com/uuid6/uuid6-ietf-draft/assets/164050/0fc59c4e-c3d8-45fe-97f5-ea9973e53583" width="100" />

You'll want the **Humanist** and **Krungthep** fonts installed for the text to render properly.  (I forget where I downloaded them from, but I'm pretty sure they're freely-available.  If not... well... I'm sure there are plenty of similar ones available and for something like this we should be using public domain fonts of some sort.)

**Comment by hi2u** [comment]
Here's a logo for UUIDv4:

![image](https://github.com/uuid6/uuid6-ietf-draft/assets/9689599/83bcb499-ef92-4291-ae3a-bd9bb91f2bba)

**Comment by sergeyprokhorenko** [comment]
One more variant
![image](https://github.com/uuid6/uuid6-ietf-draft/assets/31864557/3b1c20b1-bb2f-4c76-82aa-79d40c1e9dc6)

**Comment by sergeyprokhorenko** [comment]
![image](https://github.com/uuid6/uuid6-ietf-draft/assets/31864557/53cfbd3d-7501-4f7d-95a4-42936bf24567)

**Comment by broofa** [comment]
Keys and locks have strong associations with security and crypto.  UUIDs, [not so much](https://datatracker.ietf.org/doc/html/rfc4122#section-6).

**Comment by hi2u** [comment]
Not that I think there's likely to be some "official logo"... but, seeing we're just coming up with some ideas for fun...

Given that v7 is all bout dat timestamp... and that a "7" kinda looks like a short + long hand on an analog clock... those seem like logical + meaningful elements to incorporate into a logo.  

Maybe something like the [Aphex Twin logo](https://www.google.com/search?q=aphex+twin+logo&tbm=isch), but more clocky + computery?
